doc/source/docs/reference/geodataframe.rst

```diff
@@ -59,12 +59,20 @@ Aggregating and exploding
    GeoDataFrame.dissolve
    GeoDataFrame.explode
 
+Overlay operations
```

Comment:
@jorisvandenbossche I am not sure if this is the best header. We will have sjoin, sjoin_nearest, overlay and clip here (so far)